src/robot/libdocpkg/specbuilder.py

```diff
@@ -25,17 +26,17 @@ class SpecDocBuilder(object):
 
     def build(self, path):
         spec = self._parse_spec(path)
+
```

Comment:
We try to avoid empty lines inside functions or methods. I don't think this is needed.